### PR TITLE
Add camera section in settings and add parallax setting

### PR DESCRIPTION
**Pull request by talonateox**
hai pt.2 >.<

### CODE REVIEW

File: scripts/settings/Settings.gd [hunk 1]
```diff
@@ -76,7 +83,6 @@ const SETTINGS_CONFIG = [
 			["mouse",Setting.Type.FLOAT,1.0],
 			["absolute",Setting.Type.FLOAT,1.0]
 		]],
-		["lock",Setting.Type.BOOLEAN,true],
 		["drift",Setting.Type.BOOLEAN,false],
```
Comment (kermeow) [inline scripts/settings/Settings.gd]: this should be moved to the camera category too

File: scripts/settings/Settings.gd [hunk 2]
```diff
@@ -35,9 +40,11 @@ const SETTINGS_CONFIG = [
 		["rate",Setting.Type.FLOAT,50.0],
 		["mode",Setting.Type.ENUM,ApproachMode,ApproachMode.RATE_TIME]
 	]],
-	["parallax",Setting.Type.CATEGORY,[
-		["camera",Setting.Type.FLOAT,1.0],
-		["hud",Setting.Type.FLOAT,0.0]
+	["camera",Setting.Type.CATEGORY,[
+		["camera_parallax",Setting.Type.FLOAT,1.0],
+		["hud_parallax",Setting.Type.FLOAT,0.0],
```
Comment (kermeow) [inline scripts/settings/Settings.gd]: these two could be put into a subcategory

File: scripts/settings/Settings.gd [hunk 3]
```diff
@@ -35,9 +40,11 @@ const SETTINGS_CONFIG = [
...
+		["mode", Setting.Type.ENUM, CameraMode, CameraMode.HALF_LOCK],
```
Comment (kermeow) [inline scripts/settings/Settings.gd]: this setting and enum arent used?

**Review by kermeow — CHANGES_REQUESTED** [review]
pls read comments

File: scripts/settings/Settings.gd (hunk 3 above)
Comment (talonateox) [reply scripts/settings/Settings.gd]: you added hud parallax

Comment (kermeow) [reply scripts/settings/Settings.gd]: this is referring to the camera mode enum

**Review by kermeow — CHANGES_REQUESTED** [review]
only addressed one issue 😭😭😭

**Comment by talonateox** [comment]
I fixed them

**Review by kermeow — APPROVED** [review]

**Pull request merged**